diff --git a/crates/node_runtime/src/node_runtime.rs b/crates/node_runtime/src/node_runtime.rs
index 9ad14bd..e90dffd 100644
--- a/crates/node_runtime/src/node_runtime.rs
+++ b/crates/node_runtime/src/node_runtime.rs
@@ -373,7 +373,7 @@ impl ManagedNodeRuntime {
 
         // Note: Not in the `if !valid {}` so we can populate these for existing installations
         _ = fs::create_dir(node_dir.join("cache")).await;
-        _ = fs::write(node_dir.join("blank_user_npmrc"), []).await;
+        // _ = fs::write(node_dir.join("blank_user_npmrc"), []).await;
         _ = fs::write(node_dir.join("blank_global_npmrc"), []).await;
 
         anyhow::Ok(Box::new(ManagedNodeRuntime {
@@ -499,9 +499,9 @@ impl SystemNodeRuntime {
         let scratch_dir = paths::support_dir().join("node");
         fs::create_dir(&scratch_dir).await.ok();
         fs::create_dir(scratch_dir.join("cache")).await.ok();
-        fs::write(scratch_dir.join("blank_user_npmrc"), [])
-            .await
-            .ok();
+        // fs::write(scratch_dir.join("blank_user_npmrc"), [])
+        //     .await
+        //     .ok();
         fs::write(scratch_dir.join("blank_global_npmrc"), [])
             .await
             .ok();
